Resources/Descriptives/qml/Descriptives.qml

```diff
@@ -62,6 +62,7 @@ Form
 				}
 			}
 			CheckBox {				name: "descriptivesQQPlot";		label: qsTr("Q-Q plots")						}
+            CheckBox {				name: "descriptivesPiechart";	label: qsTr("Pie charts")						}
```

Comment:
```suggestion
			CheckBox {				name: "descriptivesPiechart";	label: qsTr("Pie charts")						}
```
rather nitpicky, QML uses tabs rather than spaces:
![afbeelding](https://user-images.githubusercontent.com/21319932/65867331-821b8b00-e376-11e9-9031-386486661534.png)

Reply:
I see, I will use tabs next time